### DISCUSSION TITLE
How AI can reply to all requests?

### DISCUSSION THREAD

**Discussion by Juste-Gnimavo** [chatwoot/chatwoot#7976] [Chatwoot APIs]
Hello,

We are in web hosting industry, we often find ourselves fielding the same questions repeatedly from our customers. To streamline our support process, we've amassed over 10,400 question-answer pairs on Chatwoot.

We're exploring the possibility of leveraging AI to address customer inquiries efficiently. Specifically, we're interested in training AI models using our extensive chatwoot database to provide prompt and accurate responses to common queries.

Additionally, we're looking into extracting user account information and product statuses (such as domain expiration dates, invoice amounts, etc.) into various file formats (SQL, PDF, Excel, etc.). This data could be instrumental in training AI models to provide even more personalized and comprehensive support.

We welcome any insights or suggestions on how to proceed with this initiative.

Thank you.

**Comment by Ghaithkhelifi** [comment]
> Hello, I'm using selfhosted chatwoot. Could I know the path of the file of OpenAI API? I would like to change the model to GPT-4
> 
> Thank you Juste

can you please show me the path i want to change the url of openai api to my api compatible with openai
